packages/formik/src/Formik.tsx

```diff
@@ -856,10 +856,12 @@ export function useFormik<Values extends FormikValues = FormikValues>({
     setValues,
     setFormikState,
     submitForm,
+
+    ...state,
   };
 
   const executeSubmit = useEventCallback(() => {
-    return onSubmit(state.values, imperativeMethods);
+    return onSubmit(state.values, imperativeMethodsAndState);
```

Comment:
I would combine them here, like `{ ...imperativeMethods, ...state }`, instead of doing it on every render.